vectordraw.py

```diff
@@ -2,6 +2,28 @@
 # script tag of the problem XML definition - see api-example.xml for
 # an example.
 
+import json
+import math
+
+
+try:
+    custom_checks
```

Comment:
@smarnach You're right, that was unnecessary. I removed the ugly `try-except` blocks and just added a comment that these can be modified by the end user directly.